tests/translator/input/api_rest_auth_disable_default_function_permissions.yaml

```diff
@@ -0,0 +1,47 @@
+Resources:
+
+  MyApiWithLambdaTokenAuth:
+    Type: AWS::Serverless::Api
+    Properties:
+      StageName: Prod
+      Auth:
+        DefaultAuthorizer: MyLambdaTokenAuth
+        Authorizers:
+          MyLambdaTokenAuth:
+            FunctionArn: !GetAtt MyAuthFn.Arn
+            DisableDefaultFunctionPermissions: true
```

Comment:
Can probably rename it to `DisableDefaultPermissions`, since in both cases (if I'm reading correctly), the context is about Lambda functions? And you could parse the property as either "disable default (function permissions)" or "disable (default function) permissions", which have a different meaning.

Reply:
Added `Function` to the name since it repeats it in the other [props](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-property-api-lambdarequestauthorizer.html) (like `FuncitonArn`, `FunctionInvokeRole` and `FunctionPayloadType`). Wouldn't it be easier for customers to connect the new property with the existing ones in the Lambda authorizer if it has `Function` in the name?

Reply:
Yeah, possible. It's pretty clear either way, so happy to keep it as-is.